desmeds/desdm_maker.py

```diff
@@ -1088,7 +1090,28 @@ def get_center(self, row, col):
         get the center location
         """
         sa = np.array(self.get_rec_shape(row, col))
-        return (sa-1.0)/2.0
+
+        # this snippet is from the piff internals
+        # it returns the central pixel of the image
+        # https://github.com/rmjarvis/Piff/blob/releases/1.2/piff/psf.py#L177
+        col_cen = int(np.ceil(col - (0.5 if sa[1] % 2 == 1 else 0)))
+        row_cen = int(np.ceil(row - (0.5 if sa[0] % 2 == 1 else 0)))
+
+        # these are the offset of the PSF position from the central pixel
+        dcol = col - col_cen
+        drow = row - row_cen
+
+        # now we add those offsets to the central pixel
+        # to get the cutout location of the PSF
+        # galsim rounds up for even images
+        # these are zero-indexed
+        row_cutout = ((sa[0] - 1)/2 if sa[0] % 2 == 1 else sa[0]/2) + drow
+        col_cutout = ((sa[1] - 1)/2 if sa[1] % 2 == 1 else sa[1]/2) + dcol
+
+        return np.array([
+            row_cutout,
+            col_cutout,
+        ])
```

Comment:
I'm not sure if you have access to the image that Piff made at the point when you are calling this function.  But this whole result should just end up being (row - image.bounds.ymin, col - image.bounds.xmin).  Maybe that's fodder for a unit test at least...

Reply:
Good point. The image isn't around which is why I am going to so much trouble here. I will check against the easy way.